src/server/routes/index.js

```diff
@@ -26,12 +22,26 @@ router.get('/signin', (request, response) => {
 router.post('/signin', (request, response) => {
   const { email, password } = request.body
   users.signIn(email, password)
-    .then((user) => {
-      if (!user) {
-        return response.send('Invalid Username/Password')
-      }
-        return response.redirect('/');
+    .then((validUser) => {
+      // if (!user) {
+      //   return response.send('Invalid Username/Password')
+      // }
+        return response.redirect(`/users/${userId}/todos`);
       });
 });
 
+router.get('/users/:id/todos', (request, response) => {
+  const { user } = request.session;
+  response.render('todo/todo-list');
+});
+
+router.post('/users/:id/todos', (request, response) => {
+  const userId = request.params.id
+  const item = request.body
+  todos.addToDo(userId, item)
+    .then((todo) => {
```

Comment:
if you doing do anything with the `todo` variable that's returned from `addToDo` then you could just not declare anything and have a empty `()`